From d709203d2f3d7621d0ed8c733712d9f176075ca6 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Tue, 15 Jan 2013 13:26:05 +0200
Subject: [PATCH 78/99] am33xx: mcasp: Add dma channel definitions

Add DMA channel definitions even though they are not used yet.
The channels still come from hwmod but it's nice to have them
in the DT too.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 arch/arm/boot/dts/am33xx.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index ed60d19..78541f8 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -582,6 +582,9 @@
 			asp-chan-q = <2>;	/* EVENTQ_2 */
 			tx-dma-offset = <0x46000000>;
 			rx-dma-offset = <0x46000000>;
+			dmas = <&edma 8
+				&edma 9>;
+			dma-names = "tx", "rx";
 		};
 
 		mcasp1: mcasp@4803C000 {
@@ -595,6 +598,9 @@
 			asp-chan-q = <2>;	/* EVENTQ_2 */
 			tx-dma-offset = <0x46400000>;
 			rx-dma-offset = <0x46400000>;
+			dmas = <&edma 10
+				&edma 11>;
+			dma-names = "tx", "rx";
 		};
 
 	};
-- 
1.8.1

